<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApiManager</class>
 <widget class="QDialog" name="ApiManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Api Manager</string>
  </property>
  <widget class="QWidget" name="teDhenatApilayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Te dhenat API</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="httpRB">
         <property name="text">
          <string>http</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="httpsRB">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string>https (*secure)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="domainTB">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>* Domain ose IP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portTB">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tryApiBTN">
         <property name="text">
          <string>Try</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ruajApiBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ruaj</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="defaultCB">
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="errorLB">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="listaApilayoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>251</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Lista API</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListView" name="listaApis"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fshiApiBTN">
      <property name="text">
       <string>Fshi Api</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
